docs/reference/builder.md

```diff
@@ -1,4 +1,5 @@
 ---
+title: Dockerfile reference
```

Comment:
Could you remove the `Dockerfile reference` title at line 18? Otherwise the documentation will generate two H1 headings